sapl/api/views.py

```diff
@@ -50,6 +53,39 @@ def recria_token(request, pk):
 
     return Response({"message": "Token recriado com sucesso!", "token": token.key})
 
+def get_mesa_diretora(request):
+    logger = logging.getLogger(__name__)
+
+    kwargs = {}
+
+    legislatura = request.GET.get('legislatura')
+    if legislatura:
+        kwargs['legislatura_id'] = legislatura
+    
+    sessao = request.GET.get('sessao')
+    if sessao:
+        kwargs['id'] = sessao
+
+    sessao_legislativa = SessaoLegislativa.objects.filter(**kwargs).order_by('-data_inicio').first()
+
+    query_legislatura = Legislatura.objects.filter(id=legislatura).order_by('-data_inicio').first()
```

Comment:
E se não for informada a legislatura?

Reply:
Vc tem certeza de que precisa dessa query???? 

`legislatura = sessao_plenaria.legislatura`

vai recuperar o objeto Legislatura